README.md

```diff
@@ -18,25 +18,25 @@ In order to build and run tests you will need a Postgres database running locall
 docker-compose up -d
 ```
 
-This will start a new, empty Postgres database in a Docker container configured as follows.
+This will start a new Postgres database initialized with Schema defined in the `schema` project, configured as follows.
 
 | Parameter | Value      | Notes                      |
 |-----------|------------|----------------------------|
 | Port      | `5432`     | This is the standard port. |
-| Database  | `gsp`      |                            |
+| Database  | `gem`      |                            |
 | User      | `postgres` |                            |
 | Password  | (none)     |                            |
 
 If you have `psql` installed locally you can connect to the database thus:
 
 ```
-psql -h localhost -U postgres -d gsp
+psql -h localhost -U postgres -d gem
```

Comment:
`gpp`?

Reply:
Well for now it's still called `gem` because that's what the downstream stuff expects. We can change it later.